Okta.Xamarin/Okta.Xamarin/Models/Item.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Okta.Xamarin.Models
+{
+    public class Item
```

Comment:
Do we need this class?

Reply:
This class is used by the Xamarin Forms views for the default shell/UI.